apps/web/dynamic_filters/column_filters.py

```
@@ -12,6 +12,7 @@ class ParticipantFilter(StringColumnFilter):
     query_param: str = "participant"
     column: str = "participant__identifier"
     label: str = "Participant"
+    or_columns: list[str] = ["participant__name"]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)